streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamTaskTest.java

```diff
@@ -1103,7 +1103,7 @@ public void shouldNotReInitializeTopologyWhenResuming() throws IOException {
         assertFalse(source1.initialized);
         assertFalse(source2.initialized);
 
-        task.resume();
+        task.resume(false);
```

Comment:
Could we also add a test case for when we call `task.resume(true)`?

Reply:
Yes, of course